txnsync/bitmask.go

```diff
@@ -110,9 +119,10 @@ func (b *bitmask) trimBitmask(entries int) {
 	default:
 	}
 
-	*b = bytes.TrimRight(*b, string(0))
+	*b = bytes.TrimRight(*b, fmt.Sprintf("%c", 0))
```

Comment:
this would have undesired performance impact. try to use "\x00" instead.

Reply:
Fixed.